File: literary_life_app/lib/models/quote.dart
class Quote {
  final int id;
  final String content;
  final String author;
  final String source;
  final String category;

  Quote({
    required this.id,
    required this.content,
    required this.author,
    this.source = '',
    this.category = '',
  });

  factory Quote.fromJson(Map<String, dynamic> json) {
    return Quote(
      id: json['id'],
      content: json['content'],
      author: json['author'] ?? '佚名',
      source: json['source'] ?? '',
      category: json['category'] ?? '',
    );
  }
}

File: literary_life_backend/app/models/quote.py
from sqlalchemy import Column, Integer, String, DateTime, func

from app.database import Base


class Quote(Base):
    __tablename__ = "quotes"

    id = Column(Integer, primary_key=True, index=True)
    content = Column(String(1000), nullable=False)
    author = Column(String(100), default="佚名")
    source = Column(String(200), default="")
    category = Column(String(50), default="文學")
    created_at = Column(DateTime(timezone=True), server_default=func.now())
